core/arch/riscv/kernel/unwind_rv.c

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*-
+ * Copyright (c) 2023 Andes Technology Corporation
+ * Copyright (c) 2015 Linaro Limited
+ * Copyright (c) 2015 The FreeBSD Foundation
+ * All rights reserved.
+ *
+ * This software was developed by Semihalf under
+ * the sponsorship of the FreeBSD Foundation.
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
could remove the full license terms text as standard BSD 2 Clause terms implicitly mention through the SPDX licence tag.

Reply:
Done